### DISCUSSION TITLE
El Salvador localization repository

### DISCUSSION THREAD

**Discussion by grottas** [OCA/odoo-community.org#110] [General]
Hi OCA team:

Can you please let me know the requirements to create the localization project? We will work in El Salvador localization and electronic invoicing.

We have been working for so long in the localization project for Colombia, not yet sent to OCA, but still working on it. You can see our work here:

https://github.com/odoo-colombia

This is our Telegram channel for this project: https://t.me/ColombiaOdoo

Now we are starting to make community with the people from El Salvador, and it would be great if the repository is available to fork it. We want are starting with this repo and now are looking with the people that has been working independently, to put together that work/code here:

https://github.com/odoo-el-salvador

This is our newly created Telegram channel for it: https://t.me/OdooSV

Suggestion for the localization repo: l10n-el-salvador

Thanks for your support, and hope to hear from you.

**Comment by pedrobaeza** [comment]
I think you can start contributing to OCA with Colombian localization, and get used to OCA conventions and flows, and then do the same with El Salvador one.
